moduleframework/tools/__init__.py

```diff
@@ -28,6 +28,7 @@
 class Basic(module_framework.AvocadoTest):
     """
     :avocado: enable
+    :avocado: tags=sanity,general,fedora,rhel
```

Comment:
I would prefer to document it, so maintainer can create tests for relevant tags. Or at least some documentation how to create and use new tags.

Reply:
It is expected that maintainer will use own tags as @bkabrda mentioned, that he is interested in own tags, like add tag like `network` to some own, to indicate and filter out that some test needs for example network available.  
These tags are now just for modulelint, and probably just: `sanity` and `optional` will be used for now. Other tags will be used more less just an example of more complex filtering abilities rather than officially document list of them somewhere.

And you are talking about how to create and use new tags, and this is avocado specific and documented well, and it is not anyhow connected with MTF: 
http://avocado-framework.readthedocs.io/en/56.0/WritingTests.html#categorizing-tests
If you will use mtf as backend, for example via behave, or unittest, it will not be anyhow used, so this help and link is not generally applicable for MTF.